src/app/screens/Accounts/index.tsx

```diff
@@ -210,12 +206,14 @@ function AccountsScreen() {
           closeTimeoutMS={200}
           isOpen={editModalIsOpen}
           onRequestClose={closeEditModal}
-          contentLabel="Edit account name"
+          contentLabel={t("edit.name_label")}
```

Comment:
According to the modal docs this is:

>   /* String indicating how the content container should be announced
     to screenreaders */}
     
Should we call it `screenreader.label`?
Should we prefix all `modal`-intrenal strings with `modal`?

Reply:
> Should we call it screenreader.label?

Makes sense!

> Should we prefix all modal-intrenal strings with modal?

We can. Currently, I'm using `label` for the text which makes the modal popup (in this case, it's in a drop-down: edit, export, delete) and then `title` for the modal title...

Reply:
Hm, yeah, not too important. Let's just do the screenreader-change and merge.

Reply:
Just came to know after working on other PRs that we can actually use tCommon for these labels. `common.actions` should have these to avoid repetition... Wdyt?

And we can just have title for strings like "Edit Account"

Reply:
For really common "words" that makes sense yeah. But not action-labels "in general" maybe?

Reply:
Oh, yes, sure, sorry!